server/src/infra/graphql/mutation.rs

```diff
@@ -440,6 +441,19 @@ impl<Handler: BackendHandler> Mutation<Handler> {
         span.in_scope(|| {
             debug!(?name, ?attribute_type, is_list, is_visible, is_editable);
         });
+        match validate_attribute_name(&name) {
+            Ok(()) => {}
+            Err(invalid_chars) => {
+                let chars = String::from_iter(invalid_chars);
+                span.in_scope(|| {
+                    debug!(
+                        "Cannot create attribute with invalid name. Invalid chars found: {}",
+                        chars
+                    )
+                });
+                return Err("Cannot create attribute with invalid name".into());
```

Comment:
Let's put the list of allowed characters there too. Ideally exposed by the common crate

Reply:
Do you want the full list, or is some "a-zA-Z" abbreviation preferred?

Reply:
Whatever is most readable. I think regex is acceptable, though it might not be clear with the dash. Maybe plain English "alphanumeric (a-z, A-Z and 0-9) and dash (-)"

Reply:
Done.